doc/metadata_routing.rst

```diff
@@ -17,16 +17,32 @@ Metadata Routing
     >>> import sklearn
     >>> sklearn.set_config(enable_metadata_routing=True)
 
-This guide demonstrates how metadata such as ``sample_weight`` can be routed
-and passed along to estimators, scorers, and CV splitters through
-meta-estimators such as :class:`~pipeline.Pipeline` and
-:class:`~model_selection.GridSearchCV`. In order to pass metadata to a method
-such as ``fit`` or ``score``, the object consuming the metadata, must *request*
-it. For estimators and splitters, this is done via ``set_*_request`` methods,
-e.g. ``set_fit_request(...)``, and for scorers this is done via the
-``set_score_request`` method. For grouped splitters such as
-:class:`~model_selection.GroupKFold`, a ``groups`` parameter is requested by
-default. This is best demonstrated by the following examples.
+This guide demonstrates how :term:`metadata`, such as ``sample_weight`` can be routed
+and passed between objects in scikit-learn. Metadata can be transfered to estimators,
+scorers, and CV splitters using :term:`meta-estimators` such as
+:class:`~pipeline.Pipeline` or :class:`~model_selection.GridSearchCV`.  It can also be
+routed between these objects without the presence of any meta-estimator. In order to
+pass metadata to a method such as ``fit`` or ``score``, the object consuming the
+metadata, must *request* it. This is done via `set_{method}_request()` methods. For
+instance, estimators and splitters would use `set_fit_request()`, and scorers would use
+the `set_score_request()` method. The following methods are supported:
```

Comment:
The following methods are not _really_ supported. Most meta-estimators don't support passing alone metadata other than for `fit` and `score`, and some for `split`. Everything else will be a long road ahead.